site/docs/extensions/filter-control.md

```diff
@@ -140,7 +126,11 @@ Dependence if you use the datepicker option: [bootstrap-datepicker](https://gith
 
 - **Detail:**
 
-   Set `input`: show an input control, `select`: show a select control, `datepicker`: show a datepicker control.
+   This options defines which kind of filter the column should have:
+   * `input`: show an input control
+   * `select`: show a select control
+   * `multipleselect`: show an advanced select control. This options has a dependency with [multiple-select](http://multiple-select.wenzhixin.net.cn/examples) library
```

Comment:
Is this used in the source or do we only need `filterControlMultipleSelect` ?

Reply:
You're right.. I need to remove that from Docs.. thanks @UtechtDustin  awesome review

Reply:
Removed